src/ray/common/id.cc

```diff
@@ -34,14 +94,51 @@ WorkerID ComputeDriverIdFromJob(const JobID &job_id) {
       std::string(reinterpret_cast<const char *>(data.data()), data.size()));
 }
 
+ObjectID ObjectID::FromPlasmaIdBinary(const std::string &from) {
+  RAY_CHECK(from.size() == kPlasmaIdSize);
+  return ObjectID::FromBinary(from.substr(0, ObjectID::LENGTH));
+}
+
+ObjectID ObjectID::GenerateActorDummyObjectId(const ActorID &actor_id) {
+  const auto dummy_task_id = TaskID::FromRandom(actor_id);
```

Comment:
I would remove this method. It is not clear when this should be called, and it's also good in this case to force the caller to consider what is happening when an actor dummy object is created.

Reply:
replied at https://github.com/ray-project/ray/pull/5286#discussion_r308887603